tutorials/OLD_custom-timescaledb-dashboards.md

```diff
@@ -405,20 +405,21 @@ You can get the full code for this project from
 
 This tutorial was originally created for HasuraCon 2021.
 
-[![Click here to watch the video](https://assets.timescale.com/docs/images/tutorials/visualizing-compression/hasuracon-talk-thumbnail.png)](https://hasura.io/events/hasura-con-2021/talks/visualizing-timescale-db-%20compression-status-in-real-time-with-hasura/ "Watch compression status in real time with Hasura")
+[![Click here to watch the video][click-here-to-watch-the-video]](https://hasura.io/events/hasura-con-2021/talks/visualizing-timescale-db-%20compression-status-in-real-time-with-hasura/ "Watch compression status in real time with Hasura")
```

Comment:
Something went wrong here. (Can we maybe get rid of those old tutorials once and for all?)